lib/get_pomo/version.rb

```diff
@@ -1,3 +1,3 @@
 module GetPomo
-  VERSION = "0.9.2"
+  VERSION = "0.9.3"
```

Comment:
undo that plz or version bump will fail ... should generally not do that in PRs

Reply:
that's what broke the travis build too ... test PR worked fine: https://github.com/grosser/get_pomo/pull/24

Reply:
Sorry, can you explain this? so the version bump is automatically? something increase automatically the version number? can you please explain me what does that?